src/rpc/rpcevo.cpp

```diff
@@ -985,7 +985,7 @@ UniValue protx_list(const JSONRPCRequest& request)
 #ifdef ENABLE_WALLET
         LOCK2(cs_main, pwallet->cs_wallet);
 
-        if (!request.params[3].isNull()) {
+        if (!request.params[4].isNull()) {
```

Comment:
Ah I see that b010602513a85424c9e5074a1b411892827b5c44 fixed the same bug as in #3259. I'd suggest to take 9b62a297bc48d0b45354932ad1133be5da10bc8b instead and merge #3259 (as it is also backportable)